proxy/bootstrap/src/main/resources/conf/server.yaml

```diff
@@ -42,19 +42,19 @@
 #  privilege:
 #    type: ALL_PERMITTED
 #
-#rules:
-#- !TRANSACTION
+#transaction:
```

Comment:
Please update the test configuration, too.
<img width="1178" alt="image" src="https://user-images.githubusercontent.com/5668787/204208912-ed15e433-a138-4bca-a670-a26c7f45e807.png">

Reply:
ok, i'll adjust it